src/components/MarketingButton/MarketingButton.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import {SocialMedia} from 'types/social-media';
+import {socialMediaUrls} from 'utils/social-media';
+import MarketingButton from './index';
+
+import YouTubeLogo from './logos/YouTubeLogo.png';
+
+describe('MarketingButton component', () => {
+  test('renders with expected props of discord as website', () => {
+    render(<MarketingButton website={SocialMedia.discord} />);
+
+    const el = screen.getByTestId('A');
+    expect(el).toBeTruthy();
+    expect(el.hasAttribute('href')).toBeTruthy();
+    expect(el.getAttribute('href')).toEqual(socialMediaUrls[SocialMedia.discord]);
+
+    expect(el.className).toContain('MarketingButton');
```

Comment:
added className test

Reply:
@denistsoi lets keep them as unit tests. Each test should ideally be testing for one thing.

Please add:

1. One test with title 'renders correctly with default classNames', which tests for the existence of `MarketingButton` class and `MarketingButton__img` class.
2. One test with the title `renders correctly with className passed in`, with the prop `className: 'test-class'`, which tests for the existence of `test-class` className and `test-class__img` className for the img.